src/bot/scripts/card.js

```diff
@@ -728,946 +66,1233 @@ const card = new Command({
-      if (hasPower || card.Name[0] === "Enforcer Jimmy") {
-
-        embed.description += "**Power Information? - Yes**\n";
-
-        for (let field in stats) {
+        if (highestToDate < 0.4) {
+          return message.channel.send("Card Not Found");
+        }
 
-          if (field === "PowerRange" && stats[field] === 0) {
+        commandValues.matchedCards = highestCard;
+        commandValues.similarity = highestToDate;
 
-            continue;
-          }
+        //const cars = ["ford-lemon",
+        //"lambo-range",
+        //"ferra-pple",
+        //"peug-rapes"];
 
-          if (field.startsWith("Power")) {
+        const url = `https://sppd.feinwaru.com/backgrounds/${commandValues.matchedCards.image}.jpg`;
 
-            embed.description += `${field === "PowerRange" ? field : camelPad(field.slice(5, field.length))}: ${typeof stats[field] === "number" ? Math.round(stats[field] * 100) / 100 : stats[field]}\n`;
-          }
-        }
+        const pathResolve = path.resolve(
+          __dirname,
+          "../assets/cards/art/",
+          commandValues.matchedCards.image + ".jpg"
+        );
 
-        if ( /*card.ChargedPowerRadius !== 0 && */ card.ChargedPowerRegen !== 0) {
+        const directoryPath = path.join(__dirname, "../assets/cards/art/");
+
+        await readDir(directoryPath)
+          .then(async files => {
+            let result = true;
+            for (let downloadfiles of files) {
+              if (
+                downloadfiles.includes(
+                  `${commandValues.matchedCards.image}.jpg`
+                )
+              ) {
+                result = false;
+              }
+            }
 
-          embed.description += `Charged Power Regen: ${Math.round(card.ChargedPowerRegen * 100) / 100}\n\n`;
-        } else {
+            if (result) {
+              async function downloadImage(pdfURL, outputFilename) {
+                let response = await rp.get({
+                    uri: pdfURL,
+                    encoding: null
+                  })
+                  .catch(console.error);;
+                fs.writeFileSync(outputFilename, response);
+              }
 
-          embed.description += "\n";
-        }
+              await downloadImage(url, pathResolve);
 
-      } else {
+            }
+          })
 
-        embed.description += "**Power Information? - No**\n\n";
-      }
+          .catch(console.error);
 
-      if (card.Type === "Character" && card.CanAttack && card.CharacterType !== "Totem") {
-        // card that can attack
+        const cardId = commandValues.matchedCards._id;
 
-        embed.description += "**Can Attack? - Yes**\n";
 
-        embed.description += `Attack Range: ${Math.round(card.AttackRange * 100) / 100}\n`;
-        embed.description += `Knockback: ${Math.round(parseInt(card.KnockbackImpulse) * 100) / 100} at ${Math.round(card.KnockbackAngleDeg * 100) / 100}°\n\n`;
+        rp(`https://sppd.feinwaru.com/api/v1/cards/${cardId}`)
```

Comment:
youre only supposed to fetch new data if the 'updated_at' fields in cards/list changes

Reply:
oh ok i wasnt aware